### DISCUSSION TITLE
Need libdelflate & libjxl update to latest version

### DISCUSSION THREAD

**Discussion by user-by** [tannerhelland/PhotoDemon#525] [News]
Please do you update to libdelflate v1.19 and libjxl v0.9.1 and later, i tried for libjxl but didnt work.

**Comment by tannerhelland** [comment]
Hi @user-by.  I will look at updating these third-party libraries in the coming weeks.

In the meantime, if you need a new libjxl immediately, download the *full* contents of the latest static Windows release:

https://github.com/libjxl/libjxl/releases/download/v0.9.1/jxl-x86-windows-static.zip

...and place all files in the /App/PhotoDemon/Plugins folder.  

If that still does not work, it means libjxl has changed their file format for the millionth time and I will need to manually investigate.  (Which is precisely why I don't update 3rd-party libraries right when a new version releases - it takes a lot of work to rewrite all interfaces to work with any new changes!)

**Reply by user-by** [reply]
Beforehand i had tried but unfortunately does not work like it, i can update to only 0.82 version. Because they had use java since 0.91

**Comment by tannerhelland** [comment]
Hello @user-by .  The latest PhotoDemon nightly builds now use the newest libjxl 0.10.0 (just released) and the latest libdeflate (1.19).  I hope they work well for you!

(libjxl changed the way they build libraries YET AGAIN, so it did take some work to rebuild PhotoDemon around the latest changes.  I hope future versions of libjxl do not require so much work between updates.  Thank you for letting me know you ran into trouble updating libjxl on your own.)
